src/main/charts/confluence/templates/configmap-jmx-config.yaml

```diff
@@ -6,5 +6,13 @@ metadata:
   labels:
     {{- include "common.labels.commonLabels" . | nindent 4 }}
 data:
-{{- include "common.jmx.config" . | nindent 2 }}
```

Comment:
Turned out having a common jmx config wasn't a very good idea. It'll be easier to push changes if we need to fix it.